### ISSUE TITLE
routes: 'UNKNOWN' is confusing

### ISSUE THREAD

**Issue by olix0r** [linkerd/linkerd2#1946]
![image](https://user-images.githubusercontent.com/240738/49601257-293bbb80-f93a-11e8-9f4b-4103275f5345.png)

When I've configured a route that encompasses all requests to a service, the default ("UNKNOWN") route receives no traffic. However, we display it prominently in the route table. Furthermore, the default sort order can place this route in the middle of my configured routes, which is a bit distracting.

Can we handle the default route specially?

I think the same applies to the CLI.

**Comment by olix0r** [comment]
The CLI uses a `[` prefix to at least cause the the route to sort separately; still I don't think it conveys helpful information:

```
:; linkerd routes deploy/voting -n emojivoto
ROUTE                                         AUTHORITY   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99   TLS
Shinola     voting-svc.emojivoto.svc.cluster.local:8080   100.00%   0.7rps           1ms           1ms    1ms    0%
Vote*       voting-svc.emojivoto.svc.cluster.local:8080   100.00%   2.3rps           1ms           3ms    4ms    0%
[UNKNOWN]   voting-svc.emojivoto.svc.cluster.local:8080     0.00%   0.0rps           0ms           0ms    0ms    0%
```

**Comment by adleong** [comment]
Do you think omitting the default route when it has 0rps would be sufficient?  I know @grampelberg had some feelings about this also.

**Comment by grampelberg** [comment]
It is pretty valuable to see that unknown is at 0rps (and therefore you've got everything handled correctly). I find it required for creating/debugging/understanding profiles.

Maybe we can pin it to the bottom? I think we're starting to run into limitations with how material allows tables to be sorted. @rmars can we provide a custom sorting function?

**Comment by wmorgan** [comment]
Not to bikeshed this further, but my 2c is that the string "[UNKNOWN]" itself looks pretty scary, and that whether we keep it in or not, I'd much prefer to see this called "(default)".

**Comment by klingerf** [comment]
The web portion of this was fixed in #1963, but we still need to update the CLI.

**Comment by adleong** [comment]
Fixed by #2066
